ui/app/styles/components/calendar-widget.scss

```diff
@@ -69,17 +67,17 @@ $dark-gray: #535f73;
       border-radius: 2px;
     }
     &.is-current-month {
-      border: 1px solid $ui-gray-900;
```

Comment:
substituted all instances of `ui-color-###` with their `readable-name` counterpart